.github/workflows/ci_workflow.yml

```diff
@@ -518,3 +534,13 @@ jobs:
         run: ctest -V
         shell: bash
         working-directory: _build
+      - name: Test CMake Consumer
+        if: matrix.build-shared == 'ON'
+        run: |
+          cmake -DCMAKE_PREFIX_PATH=../../../_install .
+          cmake --build . \
```

Comment:
I think that -DCMAKE_BUILD_TYPE=${{ matrix.build-type }} is missing; otherwise, it build a release by default. That seems much more important for Windows (refer to the discussion in issue #11530?because of some limitations/problems.

Reply:
Is there a typo in the issue number ?

Reply:
Yep, fixed.